modules/sublimeBidAdapter.js

```diff
@@ -0,0 +1,207 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+
+const BIDDER_CODE = 'sublime';
+const DEFAULT_BID_HOST = 'pbjs.ayads.co';
+const DEFAULT_SAC_HOST = 'sac.ayads.co';
+const DEFAULT_CALLBACK_NAME = 'sublime_prebid_callback';
+const DEFAULT_PROTOCOL = 'https';
+const SUBLIME_VERSION = '0.3';
+let SUBLIME_ZONE = null;
+
+/**
+ * Send a pixel to antenna
+ * @param {String} name The pixel name
+ * @param {String} [requestId]
+ */
+function sendAntennaPixel(name, requestId) {
+  if (typeof top.sublime !== 'undefined') {
+    let param = {
+      qs: {
+        z: SUBLIME_ZONE
+      }
+    };
+    if (requestId) {
+      params.qs.reqId = encodeURIComponent(requestId);
+    }
+    top.sublime.analytics.fire(name, param);
+  } else {
+    var et = Math.round(sublime.window.performance.now());
```

Comment:
Variable `sublime ` is not defined here, that's why we call manually the antenna pixel via the url

Reply:
Maybe just window.performance.now() will work ?